sos/report/__init__.py

```diff
@@ -499,7 +499,20 @@ def get_block_devs(self):
         These devices are used by add_blockdev_cmd() in the Plugin class.
         """
         try:
-            return os.listdir('/sys/block/')
+            device_list = os.listdir('/sys/block')
+            loop_devices = sos_get_command_output('losetup --all --noheadings')
+            if loop_devices['status'] == 0:
+                real_loop_devices = []
```

Comment:
This line needs to be brought outside the conditional. If `losetup` fails for whatever reason, we still need to initialize the empty list for the presence conditional lower on line 512.